JSDemos/Demos/PivotGrid/ChartIntegration/Angular/description.md

```diff
@@ -0,0 +1,44 @@
+
+If you use only the first argument and omit **integrationOptions**, default settings are used: row field values form chart [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/), and column field values form [arguments](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/). The Chart displays values of each data field on a separate [value axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/). If you need to customize the resulting chart, specify the **integrationOptions** argument.  
+
+## Convert PivotGrid Fields to Chart Series
+You can use the following **integrationOptions** to specify how PivotGrid fields should be converted to chart series:
+
+- **alternateDataFields**: Boolean       
+Specifies whether to alternate data fields in the resulting chart. This demo sets the **alternateDataFields** field to **false**.
+
+- **dataFieldsDisplayMode**: *"singleAxis"* | *"splitAxes"* | *"splitPanes"*     
+Accepts one of the following values.   
+
+    - *"singleAxis"* - values of all data fields are displayed on a single [value axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/).  
+    - *"splitAxes"* - each data field gets an individual value axis. Applies only if **putDataFieldsInto** is *"series"*.  
+    - *"splitPanes"* - each data field gets an individual [pane](/Documentation/ApiReference/UI_Components/dxChart/Configuration/panes/). Applies only if **putDataFieldsInto** is *"series"*.    
+
+    Applies only if the PivotGrid contains several [data fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/). In this demo, the **dataFieldsDisplayMode** property's value is *"splitPanes"*.
+
+- **inverted**: Boolean      
+If **true**, [row field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values go to the [chart arguments](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/) and [column field](/Documentation/Guide/UI_Components/PivotGrid/Visual_Elements/#Headers) values form [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/). If **false** - vice versa.
+
+- **putDataFieldsInto**: *"series"* | *"args"*     
+Accepts one of the following values.
+
+    - *"args"* - data fields go to the Chart arguments.  
+    - *"series"* - data fields go to the Chart series.
+
+## Advanced Customization
+The **integrationOptions** object also can contain the following functions that allow you to process PivotGrid data before visualization and customize the Chart and its series:
+
+- **processCell**       
+Processes a PivotGrid cell's data before the Chart displays it.
+
+- **customizeChart**        
+Allows you to customize the Chart's appearance.
+
+- **customizeSeries**       
+Allows you to customize the Chart's series.
```

Comment:
I have no questions about "customizeChart" because you detailed that it's all about appearance. And what do you mean by series customization? An example would be great.

Reply:
Once again, there are multiple settings related to series. These setting are different for each series type -  'area', 'bar', 'bubble', 'candlestick' , etc. But series are, in fact, part of Chart's appearance.

Reply:
I see. This is basically a subset of Chart appearance settings. I would have tried to clarify this fact, maybe a couple of examples would be in order.

Reply:
The same point as above - there is almost infinite number of ways to modify the resulting chart. It's up to developer to decide what to do.

Reply:
OK, I see. The matter is closed then.